packages/components/src/components/SquiggleViewer/index.tsx

```diff
@@ -75,27 +75,30 @@ export type SquiggleViewerProps = {
 } & PartialPlaygroundSettings;
 
 const SquiggleViewerWithoutProvider: FC<SquiggleViewerProps> = ({ value }) => {
-  const { focused } = useViewerContext();
+  const { zoomedInPath } = useViewerContext();
 
   const getSubvalueByPath = useGetSubvalueByPath();
 
-  let focusedItem: SqValue | undefined;
-  if (focused) {
-    focusedItem = getSubvalueByPath(value, focused);
+  let zoomedInItem: SqValue | undefined;
+  if (zoomedInPath) {
+    zoomedInItem = getSubvalueByPath(value, zoomedInPath);
   }
 
-  return focused ? (
+  return zoomedInPath ? (
     <div className="space-y-3 pl-3">
-      <FocusedNavigation focusedPath={focused} rootPath={value.context?.path} />
-      {focusedItem ? (
+      <ZoomedInNavigation
+        zoomedInPath={zoomedInPath}
+        rootPath={value.context?.path}
+      />
+      {zoomedInItem ? (
         <ValueViewer
-          value={focusedItem}
+          value={zoomedInItem}
           collapsible={false}
           header="large"
           size="large"
         />
       ) : (
-        <MessageAlert heading="Focused variable is not defined" />
+        <MessageAlert heading="ZoomedIn variable is not defined" />
```

Comment:
> ZoomedIn

Public message; "Zoomed in"? Revert to "Focused"?

Reply:
Will change to "Zoomed in". I think it's safer to keep the public and private interfaces similar if possible.